src/config/cfg_rules.ini

```diff
@@ -11,7 +11,8 @@ section = ifp
 section = secrets
 section = kcm
 section_re = ^secrets/users/[0-9]\+$
-section_re = ^domain/.*$
+section_re = ^domain/[^/\@]\+$
+section_re = ^domain/[^/\@]\+/[^/\@]\+$
 section_re = ^application/.*$
 
```

Comment:
I assume that "@"  and backslash "\" will be still problem for application domains because they are used as a separator in fully  qualified name by default.